### DISCUSSION TITLE
Error when trying to add a tunnel to windows server

### DISCUSSION THREAD

**Discussion by libertycircuit** [openrport/openrport#58] [General]
Hello, 

every time I try to add a tunnel for a Windows server, I get the error 'dial tcp 127.0.0.1:3389: i/o timeout'. 
Has anyone encountered this problem? If so, how can I fix the error?

**Comment by JamieMcH** [comment]
I have also encountered this issue, but I don't have a fix.
I only get the error when trying to establish an RDP connection with a remote machine, VNC works fine.
Though last time I tried to establish an RDP session a got a 'no guacd server found' error. Not sure if the 2 issues are related though.

**Comment by alhdo** [comment]
I will investigate the issue. If it's related to guacd i might have an idea why this is happening. I will keep you updated.

**Comment by Jahmax47** [comment]
I also encountered this issue. I tried enabling the Windows Remote Desktop Rule in Windows Firewall and it worked just fine. Just open up PowerShell as administrator and run the following cmdlet:

Enable-NetFirewallRule -DisplayGroup "Remote Desktop"

Cheers
